target/linux/ath79/patches-5.10/401-mtd-nor-support-mtd-name-from-device-tree.patch

```diff
@@ -0,0 +1,51 @@
+From f32bc2aa01edcba2f2ed5db151cf183eac9ef919 Mon Sep 17 00:00:00 2001
```

Comment:
Are you taking the bootargs from U-Boot and do not overwrite them in the OpenWrt device tree?
If you nee the patch which adds the mtd name because U-Boot uses a specific mtd name in the bootargs, then please add this to the commit description.

Reply:
Yes, I have now added an extra paragraph about this to the commit message.